app/src/main/java/org/thoughtcrime/securesms/onboarding/LandingActivity.kt

```diff
@@ -19,28 +53,128 @@ class LandingActivity : BaseActionBarActivity() {
         // Session then close this activity to resume the last activity from the previous instance.
         if (!isTaskRoot) { finish(); return }
 
-        val binding = ActivityLandingBinding.inflate(layoutInflater)
-        setContentView(binding.root)
         setUpActionBarSessionLogo(true)
-        with(binding) {
-            fakeChatView.startAnimating()
-            registerButton.setOnClickListener { register() }
-            restoreButton.setOnClickListener { link() }
-            linkButton.setOnClickListener { link() }
-        }
+
+        ComposeView(this)
+            .apply { setContent { AppTheme { LandingScreen() } } }
+            .let(::setContentView)
+
         IdentityKeyUtil.generateIdentityKeyPair(this)
         TextSecurePreferences.setPasswordDisabled(this, true)
         // AC: This is a temporary workaround to trick the old code that the screen is unlocked.
         KeyCachingService.setMasterSecret(applicationContext, Object())
     }
 
-    private fun register() {
-        val intent = Intent(this, RegisterActivity::class.java)
-        push(intent)
+    @Preview
+    @Composable
+    private fun LandingScreen(
+        @PreviewParameter(ThemeResPreviewParameterProvider::class) themeResId: Int
+    ) {
+        PreviewTheme(themeResId) {
+            LandingScreen()
+        }
+    }
+
+    @Composable
+    private fun LandingScreen() {
+        Column(modifier = Modifier.padding(horizontal = 36.dp)) {
+            Spacer(modifier = Modifier.weight(1f))
+            Text(stringResource(R.string.onboardingBubblePrivacyInYourPocket), modifier = Modifier.align(Alignment.CenterHorizontally), style = MaterialTheme.typography.h4, textAlign = TextAlign.Center)
+            Spacer(modifier = Modifier.height(24.dp))
```

Comment:
Would we want some of the heights and paddings and things to be constants so that we can adjust them all in one go if we wanted? So rather than `14.dp` we might have `lineItemPadding` or such?

Reply:
It's a good idea, however I didn't really follow the rhyme or reason of the designs, so I'd be shoehorning a semantic probably 1) incorrectly and maybe 2) prematurely, so I'd prefer to just keep it simple at the moment until it's a bit clearer.

Reply:
Fair enough - marking resolved.